src/MoreOrLess/makeStories.js

```diff
@@ -0,0 +1,47 @@
+import React, { Fragment } from 'react';
+
+const LoremIpsum = () => (
+  <div>
```

Comment:
Replace this with Flex accepted as an argument, so that it'll work cross platform, or just make LoremIpsum a string const and use it in your story like `{loremIpsum}`, no need to make it a react component :)